build.gradle.kts

```diff
@@ -39,6 +39,7 @@ dependencies {
     api("com.atlassian.performance.tools:jira-actions:[2.0.0,4.0.0)")
     api("com.atlassian.performance.tools:ssh:[2.0.0,3.0.0)")
     api("com.atlassian.performance.tools:virtual-users:[3.3.0,4.0.0)")
+    api("com.amazonaws:aws-java-sdk-ec2:1.11.424")
```

Comment:
This doesn't affect dependency locks, but it does affect the released POM generated by `./gradlew generatePom`